test.hxml

```diff
@@ -0,0 +1,14 @@
+## Neko
+-main TestMain
+-lib munit
+-lib hamcrest
+-lib svg
+-lib openfl
+-lib lime
+
+-cp src
+-cp test
+-neko build/neko_test.n
+-D lime
+-D lime_cairo
+--remap flash:openfl
```

Comment:
Do you mean, instead of this `##Neko ...` block, I just need to add `-cmd neko build/neko_test.n`?